### PR TITLE
Bug 1226583 - [tc-gip] Tests that requires dogfood build fails, because they run on regular eng builds

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59c64bf24c58166361c1a1603e44b5a896b592fe

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_apps_present.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestAppsArePresent(GaiaTestCase):
+
+    def test_spark_apps_are_present(self):
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_apps_present.py]: I guess the spark name can be removed here then too, right?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/bugzillalite/manifest.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[parent:../manifest.ini]
+
+[DEFAULT]
+external = true
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/bugzillalite/manifest.ini]: external = true by default, see: http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini#54
So this is not really necessary, it seems.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_apps_present.py (hunk 1 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_apps_present.py]: That's right. Fixed.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/bugzillalite/manifest.ini (hunk 2 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/bugzillalite/manifest.ini]: Good catch. Fixed

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c4f5fdcf7f9f18c8bcef5d8ed0e0310767fff670

**Pull request merged**